### ISSUE TITLE
how to improve the resolution of generated image based on med-ddpm

### ISSUE THREAD

**Issue by leeh2213** [mobaidoctor/med-ddpm#35]
Thank you for the excellent work on Med-DDPM! I have successfully trained a Med-DDPM model using my own data, but I've noticed that the resolution of the generated images is relatively low. Could you please provide any suggestions on how to improve the image resolution based on the trained Med-DDPM model without needing to retrain from scratch?

**Comment by mobaidoctor** [comment]
Hi @leeh2213, Thank you for your interest in our work and for your inquiry. We're pleased to hear that you trained our method on your dataset. If the generated images have low resolution, it likely indicates that the majority of images in your dataset were of low resolution. The easiest solution would be to subselect the high-resolution images from your dataset and fine-tune your pretrained model on this subset with a smaller learning rate. This approach should help adjust your model to generate higher-resolution outputs.  If you have any further questions or need clarification, please don't hesitate to let us know. Thank you

**Comment by leeh2213** [comment]
Thank you for your advice. I'm already trying a similar method, as you mentioned, and I'm currently waiting for the results. Additionally, I'm experimenting with some super-resolution methods (specifically, another 2D diffusion model) using 2D slice images from the z-axis of the generated image to enhance the quality of the low-resolution outputs. This approach allows me to generate higher-resolution images along the z-axis, but the resolution remains low in the other two axes.
